tools/osbuild-image-test

```diff
@@ -148,19 +150,23 @@ class TestCase:
     can dump the built image-info to disk
     """
 
-    def __init__(self, path, data: Dict, osb: OSBuild, imi: ImageInfo) -> None:
+    built_image_info: ImageInfo
```

Comment:
nitpick: there's AFAIK no benefit in defining the variable on the class level, since it is defined in the constructor. Also you are not doing it for any other variables.

Reply:
The thing is, `built_image_info` can't be defined in the constructor. So it's type would have to be an `Optional[ImageInfo]` type. Setting the variable at class level allows me to not face the error `attribute first seen outside of init` (or something equivalent) and also allows me to avoid aving to deal with a field that can be `Optional` Because I would have to include checks. But in fairness, adding checks could be a good idea too!

Reply:
I don't see how this could happen, since you always assign `ImageInfo()` instance to it in the constructor 🤔

Reply:
I'm settling for initializing an empty one.